config/env.config.js

```diff
@@ -55,5 +57,77 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
```

Comment:
Use lambda's instead unless you absolutely need the `this` to reference the `window/executing function`